.all-contributorsrc

```diff
@@ -72,6 +72,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "Gandem",
+      "name": "Gandem",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/11560964?v=3",
+      "profile": "https://github.com/Gandem",
+      "contributions": [
+        "code"
```

Comment:
Want to add `test` to this as well? 😉  Run `npm start contrib.gen` after you update this :+1: If you want to :)